src/topp/DecoyDatabase.cpp

```diff
@@ -165,173 +187,215 @@ class TOPPDecoyDatabase :
     f.writeStart(out);
     FASTAFile::FASTAEntry entry;
 
-    // Configure Enzymatic digestion
-    // TODO: allow user-specified regex
-    ProteaseDigestion digestion;
-    String enzyme = getStringOption_("enzyme").trim();
-    if ((input_type == SeqType::protein) && !enzyme.empty())
-    {
-      digestion.setEnzyme(enzyme);
-    }
-    // check if decoy_string is common decoy string (e.g. decoy, rev, ...)
-    String decoy_string_lower = decoy_string;
-    decoy_string_lower.toLower();
-    bool is_common = false;
-    for (const auto& a : DecoyHelper::affixes)
+
+ // Neighbor peptide search   
+    String neighbor_file = getStringOption_("Neighbor_Search:neighbor_in");
+ // Check if the neighbor file option is provided
```

Comment:
don't state the obvious, but the intent.
Here:
`// create neighbor peptides for the relevant peptides given in the FASTA from '-in'`